EKrut-Server/src/ekrut/server/db/OrderDAO.java

```diff
@@ -260,21 +262,27 @@ public boolean updateOrderStatus(int orderId, OrderStatus status) {
 	
 	
 	// For Almog: Added by Nir, get list of orders in Shipment status.
-	public ArrayList<Order> fetchOrderShipmentList(String area){
-		PreparedStatement ps = con.getPreparedStatement("SELECT orderId FROM orders WHERE type = 'SHIPMENT' AND area = ?");
+	public ArrayList<Order> fetchOrderShipmentListByArea(String area){
+		PreparedStatement ps = con.getPreparedStatement(
+				"SELECT orderId, date, status, type, dueDate, clientAddress, location, username"
+				+ "FROM orders"
+				+ "WHERE type = 'SHIPMENT'");
 		ArrayList<Order> orderList = new ArrayList<>();
 		try {
 			ps.setString(1, area);
```

Comment:
The new query doesn't have any parameters.

Reply:
Fixed